mlxtend/evaluate/tests/test_time_series.py

```diff
@@ -0,0 +1,260 @@
+# mlxtend Machine Learning Library Extensions
+#
+# Time series cross validation with grouping.
+# Author: Dmitry Labazkin <labdmitriy@gmail.com>
+#
+# License: BSD 3 clause
+
+import numpy as np
+import pytest
+from mlxtend.evaluate import GroupTimeSeriesSplit
+
```

Comment:
The whole unit test suite here looks pretty comprehensive to me. I wonder if we could add a computationally cheap scikit-learn-related test though. E.g., plugging it into `cross_val_score` or `GridSearchCV` as `cv` argument?

Reply:
Added test to check the usage with cross_val_score based on DummyClassifier with "most_frequent" strategy.
So with this split we will have 3 splits and the following values/accuracy for test true and predicted targets:

|       y      |   y_pred  | accuracy |
| --------- | --------- | ---------- |
|    [1 1]    |    [0 0]    |       0       |
|    [0 1]    |    [1 1]    |      0.5     |
| [1 0 0 0] | [1 1 1 1] |     0.25    |

Reply:
This is nice, thanks! Mainly why I suggested it is to check for API compatibility. I am pretty sure the it is compatible with `GridSearchCV` and `cross_val_score`, but you never know. It's also to make sure in case we modify it in the future that it still works, or in case the scikit-learn API changes, it still works. 

What I had in mind was something like:

```
from mlxtend.evaluate import GroupTimeSeriesSplit
from sklearn.linear_model import LogisticRegression
from sklearn.model_selection import cross_val_score

lr = LogisticRegression()

cv = GroupTimeSeriesSplit(...)
cross_val_score(lr, X, y, cv=cv)
```